plugins/get_url/komand_get_url/actions/get_file/action.py

```diff
@@ -15,69 +13,45 @@ def __init__(self):
             input=GetFileInput(),
             output=GetFileOutput(),
         )
+        self.utils = Utils(action=self)
 
     def run(self, params={}):
-        utils = Utils(action=self)
         url = params.get(Input.URL)
         checksum = params.get(Input.CHECKSUM)
-        tout = params.get(Input.TIMEOUT, 60)
+        timeout = params.get(Input.TIMEOUT, 60)
         is_verify = params.get(Input.IS_VERIFY, True)
-
-        # Check for supported url prefix
-        utils.validate_url(url)
-
-        meta = utils.hash_url(url)
-        cache_file = "/var/cache/" + meta["file"]
-
-        # Attempt to retrieve headers from past request
-        headers = {}
-        if komand.helper.check_cachefile(meta["metafile"]):
-            headers = utils.check_url_meta_file(meta)
-
-        # Download file
-        url_object = komand.helper.open_url(
-            url,
-            timeout=tout,
-            verify=is_verify,
-            If_None_Match=headers.get("etag", ""),
-            If_Modified_Since=headers.get("last-modified", ""),
-            User_Agent=params.get(Input.USER_AGENT, "Mozilla/5.0"),
-        )
-
+        user_agent = params.get(Input.USER_AGENT, "Mozilla/5.0")
```

Comment:
I would really like it if we could replace all the instances of the hardcoded user agent string `"Mozilla/5.0"` with a constant variable, maybe in `plugins/get_url/komand_get_url/__init__.py` or some other place for constants. That way if we ever need to change it, we can simply change it in one place and it will take effect everywhere at once.

Reply:
Would be awesome if we could also make the timeout `60` on line 21 into a `DEFAULT_TIMEOUT = 60` too, but not necessary.